### DISCUSSION TITLE
Code of Conduct review for Maintainers and Users.

### DISCUSSION THREAD

**Discussion by Alex313031** [Alex313031/thorium#30] [Announcements]
@gz83 @midzer Please read/review the new Thorium Code of Conduct, and if you agree, respond below with "I Agree".

https://github.com/Alex313031/Thorium/blob/main/CODE_OF_CONDUCT.md

While I don't feel that you guys NEED to read this, because you're both nice people, I am making it a requirement going forward for all users and maintainers of Thorium that interact here on GitHub or alex313031.blogspot.com or any future Thorium website.

**Comment by midzer** [comment]
For me a CoC is more or less "common human sense". But yeah, I agree.

**Reply by Alex313031** [reply]
@midzer Lol I got the idea from https://github.com/iridium-browser/iridium-browser/blob/master/CODE_OF_CONDUCT.md
which basically says that. But I wanted a more proper/official one, so I copied the one in the root of the chromium tree, and then added to it and modified some stuff.

**Comment by gz83** [comment]
> @gz83 @midzer Please read/review the new Thorium Code of Conduct, and if you agree, respond below with "I Agree".
> 
> https://github.com/Alex313031/Thorium/blob/main/CODE_OF_CONDUCT.md
> 
> While I don't feel that you guys NEED to read this, because you're both nice people, I am making it a requirement going forward for all users and maintainers of Thorium that interact here on GitHub or alex313031.blogspot.com or any future Thorium website.

@Alex313031 I Agree.
I have read all the content of COC, I think it can make Thorium project more standardized.

**Comment by Alex313031** [comment]
@midzer @gz83 Marking as completed, but leaving open in case anyone decides to be an asshole and that way I can refer them here.

**Comment by Alex313031** [comment]
@midzer @gz83 I made minor fixes to the COC. Also, it is a new year. SO can ya'll simply let me know you still agree to the COC.

You don't have to read it all again. I just want there to be accountability.

For example, I don't want more people to join the team in the future, and then have someone complain "but you didn't make "them" re-agree, so why do I have to?"

**Reply by gz83** [reply]
I agree with everything in COC

**Comment by Alex313031** [comment]
@midzer @gz83 Attention Thorium contributors:

It is a new year!! So:

1. I re-added you all as collaborators to the repos, please accept the invitations.
2. We need to update all the copyrights in all files
3. The Code of Conduct > https://thorium.rocks/CoC.html has not changed, but since it is a new year, just like [last time](https://github.com/Alex313031/thorium/discussions/30#discussioncomment-6707748), I am requiring you all to comment below stating whether you agree or disagree with the CoC. This is for accountability, posterity, and so that any future collaborators can't say "but you didn't make \**insert name here*\* agree, so why do *I* have to agree?"

**Reply by midzer** [reply]
1) okay, thanks
2) This should be perfectly doable via search & replace (in an IDE with preview). So that you just confirm each individual replacement. I can do it if noone else wants to do.
3) As mentioned the last time, I am no big "fan" of CoC, but to keep community together and not start a discussion about it, I  agree wholeheartedly

**Reply by gz83** [reply]
I agree with everything and can help with any reasonable need

@Alex313031
